src/forcegraph-kapsule.js

```diff
@@ -207,7 +208,11 @@ export default Kapsule({
       //
 
       function layoutTick() {
-        if (++state.cntTicks > state.cooldownTicks || (new Date()) - state.startTickTime > state.cooldownTime) {
+        if (
+          ++state.cntTicks > state.cooldownTicks ||
+          (new Date()) - state.startTickTime > state.cooldownTime ||
+          state.d3ForceLayout.alpha() < state.d3AlphaMin
```

Comment:
As mentioned previously we need to check whether it's using the D3 layout (`isD3Sim`) before checking this condition.